pyccel/parser/semantic.py

```diff
@@ -1132,6 +1136,73 @@ def _handle_function(self, expr, func, args, *, is_method = False, use_build_fun
                             func, func.is_elemental)
 
             return new_expr
+    def _handle_kernel(self, expr, func, args, **settings):
+        """
+        Create the node representing the kernel function call.
+
+        Create a FunctionCall or an instance of a PyccelInternalFunction
+        from the function information and arguments.
+
+        Parameters
+        ----------
+        expr : IndexedFunctionCall
+               Node has all the information about the function call
+
+        func : FunctionDef instance, Interface instance or PyccelInternalFunction type
+               The function being called.
+
+        args : tuple
```

Comment:
Wrong type

Reply:
fixed 1b5ad2e

Reply:
Type is still wrong. Your docstring says that the type should be `list[LiteralInteger]`

Reply:
can u check now please https://github.com/pyccel/pyccel-cuda/commit/adad61a1dbe0f79e7e321632defcff2e708f0d11

Reply:
It is now a complete type :+1:

Reply:
This fix seems to have been reverted. We are back to a wrong incomplete type

Reply:
[07cf232](https://github.com/pyccel/pyccel-cuda/pull/42/commits/07cf23209cb7d8ad74090fa3249cd5dd111f1193)

Reply:
This commit doesn't contain changes on line 1161 of parser/semantic.py

Reply:
[84c1b53](https://github.com/pyccel/pyccel-cuda/pull/42/commits/84c1b535cf9ce3a5c5cfc204c2d9da5abdd98843)